Kickstarter-iOS/Views/Cells/SettingsCurrencyCell.swift

```diff
@@ -22,11 +24,17 @@ final class SettingsCurrencyCell: UITableViewCell, NibLoading, ValueCell {
     _ = self.titleLabel
       |> settingsTitleLabelStyle
 
+    _ = self.chevronImageView
```

Comment:
Wonder if we should use the same right padding for both the `..WarningCell` and this cell?

<img width="509" alt="screen shot 2019-02-04 at 12 07 31 pm" src="https://user-images.githubusercontent.com/387596/52234073-a4a3e580-2875-11e9-9e9c-fddb8d18c1ec.png">